test/sequential/test-inspector-esm.js

```diff
@@ -0,0 +1,153 @@
+'use strict';
+const common = require('../common');
+
+common.skipIfInspectorDisabled();
+
+const assert = require('assert');
+const fixtures = require('../common/fixtures');
+const { NodeInstance } = require('../common/inspector-helper.js');
+
+function checkListResponse(response) {
+  assert.strictEqual(1, response.length);
+  assert.ok(response[0]['devtoolsFrontendUrl']);
+  assert.ok(
+    /ws:\/\/127\.0\.0\.1:\d+\/[0-9A-Fa-f]{8}-/
+      .test(response[0]['webSocketDebuggerUrl']));
+}
+
+function checkVersion(response) {
+  assert.ok(response);
+  const expected = {
+    'Browser': `node.js/${process.version}`,
+    'Protocol-Version': '1.1',
+  };
+  assert.strictEqual(JSON.stringify(response),
+                     JSON.stringify(expected));
+}
+
+function checkBadPath(err) {
+  assert(err instanceof SyntaxError);
+  assert(/Unexpected token/.test(err.message), err.message);
```

Comment:
This error comes from the engine, not Node, right? If so, we shouldn't validate the `message`, as it could be different from V8 to Chakra.

Reply:
This was a hardcoded string - but there's no reason to test this here, it is covered by the main inspector test. I removed it.